Source/ProjectRimFactory/Common/HarmonyPatches/UpdateResourceCountsPatch.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using RimWorld;
+using Verse;
+using UnityEngine;
+using HarmonyLib;
+using System.Threading.Tasks;
+
+namespace ProjectRimFactory.Common.HarmonyPatches
+{
+
+
+    [StaticConstructorOnStartup]
+    public static class AssemblerQueueManager
+    {
+        public static List<IAssemblerQueue> AssemblerQueue = new List<IAssemblerQueue>();
```

Comment:
This won't reset on game restart or anything like that.

Prefer to add the AssemblerQueue to the PRFGameComponent as a local variable - that way, it's tied to the specific game.